packages/documentation/docs/api/vuefire.md

```diff
@@ -5,11 +5,15 @@ For all code samples, we will consider a `db` variable is imported as follows:
 <FirebaseExample id="db creation">
 
 ```js
+import firebase from 'firebase'
```

Comment:
It should be a more specific import of `firebase/app`

Reply:
Hi, I couldn't get my app working with simply `import firebase from "firebase/app"`
I had to also include `import "firebase/firestore"`. Would like to learn more about how this works as I am pretty new to JS.

Reply:
yeah, you need to import multiple things, you should be able to find detailed explanations on Firebase docs

Reply:
Cool. I looked at the source code and importing the subpackages essentially calls App.registerService which hooks into vue ecosystem. https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/index.ts#L29

anyways I think I made all the changes. I put import firestore and import database where required in the docs.